conekta_cash_gateway.php

```diff
@@ -86,21 +86,35 @@ public function ckpg_webhook_handler()
         header('HTTP/1.1 200 OK');
         $body          = @file_get_contents('php://input');
         $event         = json_decode($body, true);
+
+         // Respondiendo a eventos "ping"
+         if (isset($event['type']) && $event['type'] === 'webhook_ping') {
+            header('Content-Type: application/json');
+            echo json_encode(['message' => 'OK']);
+            exit; // Salir de la función después de manejar el evento ping
```

Comment:
exit ó return ?

Reply:
https://stackoverflow.com/questions/3484050/php-exit-or-return-which-is-better

Reply:
xD